.editorconfig

```diff
@@ -11,10 +11,13 @@ indent_size = 2
 insert_final_newline = true
 trim_trailing_whitespace = true
 
+[*.ts]
+quote_type = single
+
+
```

Comment:
Interesting, I'm not sure where I saw this and copied it from. It seems like a nice idea to tell the editor what type of quotes to use when generating code since we have seen VS Code guess wrong in our TS files before. I'm sorta of the opinion that it doesn't hurt to leave it there incase it is ever implemented.

Reply:
Sure, I can see that, especially because it seems like there are tools out there that support it. 🙂